package-lock.json

```diff
@@ -5,7 +5,7 @@
   "requires": true,
```

Comment:
please remove this file from the pull request

Reply:
Should this actually be in .gitignore?

Reply:
No, the file is used by the CI to check whether new packages have been installed, in which case the build time will be much longer. But the file should only be committed if intentionally a new package has been installed or updated. Its a bit annoying that running `npm install` always changes something in this package, `npm ci` can be used to install the exact contents of the `package-lock.json` but for some users that resulted in problems.